src/main/java/com/spotinst/sdkjava/model/api/azure/statefulNode/ApiStatefulNodeAttachDataDiskConfiguration.java

```diff
@@ -0,0 +1,120 @@
+package com.spotinst.sdkjava.model.api.azure.statefulNode;
+
+import com.fasterxml.jackson.annotation.JsonFilter;
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.spotinst.sdkjava.client.rest.IPartialUpdateEntity;
+
+import java.util.HashSet;
+import java.util.Set;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonFilter("PartialUpdateEntityFilter")
+public class ApiStatefulNodeAttachDataDiskConfiguration implements IPartialUpdateEntity {
+
+    @JsonIgnore
+    private Set<String> isSet;
+    private String          dataDiskName;
+    private String          dataDiskResourceGroupName;
+    private Integer         lun;
+    private Integer         sizeGB;
+    private String          storageAccountType;
```

Comment:
Should this field be enumeration?

Reply:
common enum "AzureStatefulNodeDiskTypeEnum" file added for both os and data disk types